service/grpc/codelingo/codelingo.pb.go

```diff
@@ -1041,34 +705,6 @@ func (m *Position) String() string            { return proto.CompactTextString(m
 func (*Position) ProtoMessage()               {}
 func (*Position) Descriptor() ([]byte, []int) { return fileDescriptor0, []int{21} }
 
-func (m *Position) GetFilename() string {
-	if m != nil {
-		return m.Filename
-	}
-	return ""
-}
-
-func (m *Position) GetOffset() int64 {
-	if m != nil {
-		return m.Offset
-	}
-	return 0
-}
-
-func (m *Position) GetLine() int64 {
-	if m != nil {
-		return m.Line
-	}
-	return 0
-}
-
-func (m *Position) GetColumn() int64 {
-	if m != nil {
-		return m.Column
-	}
-	return 0
-}
-
 type LatestClientVersionRequest struct {
 }
 
```

Comment:
exported method LatestClientVersionRequest.Reset should have comment or be unexported